src/html.rs

```diff
@@ -167,12 +169,42 @@ pub fn embed_srcset(
     options: &Options,
 ) -> String {
     let mut array: Vec<SrcSetItem> = vec![];
-    let re = Regex::new(r",\s+").unwrap();
-    for srcset_item in re.split(srcset) {
-        let parts: Vec<&str> = srcset_item.trim().split_whitespace().collect();
-        if parts.len() > 0 {
-            let path = parts[0].trim();
-            let descriptor = if parts.len() > 1 { parts[1].trim() } else { "" };
+
+    // Parse srcset attribute according to the specs
+    // https://html.spec.whatwg.org/multipage/images.html#srcset-attribute
+    let mut offset = 0;
+    let size = srcset.chars().count();
+
+    while offset < size {
+        let mut has_descriptor = true;
+        // Zero or more ASCII whitespace + skip leading U+002C COMMA character (,)
```

Comment:
Just "comma" should be enough.  And "ASCII whitespace" is too specific, we have those whitespaces defined in the array, we have that part described in the file already, just "whitespace" would be easier to read.

Something like this:

> Zero or more whitespaces + skip leading comma